specification/vmware/resource-manager/Microsoft.AVS/AVS/clusters.tsp

```diff
@@ -102,7 +102,7 @@ model ClusterUpdate {
 
   #suppress "@azure-tools/typespec-azure-core/no-legacy-usage" "Legacy flatten"
   @doc("The properties of a cluster resource that may be updated")
-  @Azure.ClientGenerator.Core.Legacy.flattenProperty
+  @Azure.ClientGenerator.Core.Legacy.flattenProperty("!javascript")
```

Comment:
Generally these decorators should be in `client.tsp` file and this would be easy for us to understand this is only client behavior and no need for ARM review.

@josefree Do we have any agreement that no need for `ARMReview` if any changes in `client.tsp` ?